Monika-After-Story/MonikaModDev
# Docking station

owo whats this?

its a way to enable a "Docking station" for easy file in/out in base64. Now we can make giving monika items and having monika give you items way easier.

### Testing
* make sure each function works lol

To test the main functionality:
1. use `randomblob` from mas_utils to generate a blob of random data in a StringIO buffer
2. call `sendPackage` on `mas_docking_station` to send that blob of data into the characters folder.
    * this will be like `checksum = mas_docking_station.sendPackage("packagename", blobofdata, True, True)`
3. call `signForPackage` on `mas_docking_station` to check that data and reference it
    * `mas_docking_station.signForPackage("packagename", checksum)`
    * this should return True if the checksum matched
    * if `keep_contents` is False, then the package will be deleted if checksums matched

There are other functions, but I have not tested because 12% pudding stout and omg omgomgomgomgomgomgomgogogogmogmgogmogmog idk too excited for no reason.

Read documentation, fuck bitches
